api/v1alpha1/capacity_types.go

```diff
@@ -238,13 +238,15 @@ type Quota struct {
 type CapacityStatus struct {
 	// usedResources are the resources (by flavor) currently in use by the
 	// workloads assigned to this capacity
-	UsedResources map[corev1.ResourceName]map[string]Usage `json:"usedResources,omitempty"`
+	UsedResources ResourceUsage `json:"usedResources,omitempty"`
 
 	// assignedWorkloads is the number of workloads currently assigned to this
 	// capacity.
 	AssignedWorkloads int32 `json:"assignedWorkloads,omitempty"`
 }
 
+type ResourceUsage map[corev1.ResourceName]map[string]Usage
```

Comment:
Same as the variable name, UsedResources? 241 reads better if we do that unless we think this will be used somewhere else

Reply:
Yeah, it makes sense.